galaxy/values-cvmfs.yaml

```diff
@@ -95,7 +108,12 @@ extraFileMappings:
               </div>
           </body>
       </html>
-
+  /tmp/probedb.py:
```

Comment:
This should be added to non-cvmfs values file as well if we keep it in values. Alternatively, we could add it in directly in the template since it's essential to the probes?

Reply:
+1 to @almahmoud's suggestion. We have a limited budget for the length of values.yaml, so it would be better to include this as a permanent fixture, maybe in this file: https://github.com/galaxyproject/galaxy-helm/blob/5e2508beb87b973ebaa282f5fb9fbd60ab137b9b/galaxy/templates/configmap-extra-files.yaml#L21

Reply:
suggested revision made in [configmap-extra-files.yaml](https://github.com/luke-c-sargent/galaxy-helm/blob/10e20ed149970e0ab6bdca95192726cca26b4262/galaxy/templates/configmap-extra-files.yaml#L25-L37), with a deployment-job.yaml [volumeMount](https://github.com/luke-c-sargent/galaxy-helm/blob/10e20ed149970e0ab6bdca95192726cca26b4262/galaxy/templates/deployment-job.yaml#L117-L119) and corresponding [volume](https://github.com/luke-c-sargent/galaxy-helm/blob/10e20ed149970e0ab6bdca95192726cca26b4262/galaxy/templates/deployment-job.yaml#L151-L153)